### ISSUE TITLE
prisma2 dev failed with sqlite database

### ISSUE THREAD

**Issue by rakeshrana15** [prisma/prisma2#1201]
Prisma2 dev failed with sqlite database on windows

Error: Command failed with exit code 255: C:\Users\krakes\AppData\Roaming\npm\node_modules\prisma2\migration-engine.exe cli --datasource sqlite:// --create_database

![prisma2 dev error](https://user-images.githubusercontent.com/20044916/71307610-8e176600-2416-11ea-9d1f-9ad3de3911d1.png)

**Comment by janpio** [comment]
Thanks for creating this issue @rakeshrana15.

Can you share your `schema.prisma`, or at least the datasource part of it? 
How did you create that file - manually or via `prisma2 init`?

**Comment by rakeshrana15** [comment]
I used existing mysql database with 2 independent tables. tried this with sqlite db also. I wanted to generate the boilerplate for the same, however i could only generate schema.prisma as below.

-------------Schema.Prisma --------------------------------------------------------------------------------
generator photon {
  provider = "photonjs"
}

datasource db {
  provider = "mysql"
  url      = "mysql://root:root@127.0.0.1/testdb"
}

model profile {
  AccountNo        Int?
  Active           String?
  Company          String?
  ContactNo        Int?
  DropPoint        String?
  DropTime         String?
  Email            String?
  EmergencyContact Int?
  Gender           String?
  HomeAddress      String?
  Name             String?
  OfficeAddress    String?
  PickupPoint      String?
  PickupTime       String?
  ProfileID        Int     @id
  ProfilePic       String?
  Rank             Int?
  User             String?
  VehicalOwner     String?
}

model users {
  Email        String?
  id           Int     @id
  Password     String?
  PasswordHash Int?
  RealName     String?
  RoleID       String?
  User         String?
}
______________________________________________
-------Database tables ---------------
CREATE TABLE `profile` (
  `ProfileID` int(11) NOT NULL,
  `AccountNo` int(11) DEFAULT NULL,
  `Active` varchar(191) DEFAULT NULL,
  `Company` varchar(191) DEFAULT NULL,
  `ContactNo` int(11) DEFAULT NULL,
  `DropPoint` varchar(191) DEFAULT NULL,
  `DropTime` varchar(191) DEFAULT NULL,
  `Email` varchar(191) DEFAULT NULL,
  `EmergencyContact` int(11) DEFAULT NULL,
  `Gender` varchar(191) DEFAULT NULL,
  `HomeAddress` varchar(191) DEFAULT NULL,
  `Name` varchar(191) DEFAULT NULL,
  `OfficeAddress` varchar(191) DEFAULT NULL,
  `PickupPoint` varchar(191) DEFAULT NULL,
  `PickupTime` varchar(191) DEFAULT NULL,
  `ProfilePic` varchar(191) DEFAULT NULL,
  `Rank` int(11) DEFAULT NULL,
  `User` varchar(191) DEFAULT NULL,
  `VehicalOwner` varchar(191) DEFAULT NULL,
  PRIMARY KEY (`ProfileID`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8

CREATE TABLE `users` (
  `id` int(11) NOT NULL,
  `Email` varchar(191) DEFAULT NULL,
  `Password` varchar(191) DEFAULT NULL,
  `PasswordHash` int(11) DEFAULT NULL,
  `RealName` varchar(191) DEFAULT NULL,
  `RoleID` varchar(191) DEFAULT NULL,
  `User` varchar(191) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8

______________________________________

**Comment by janpio** [comment]
Can you reproduce the error when you start a new project again? I can not imagine how your `schema.prisma` could lead to the error message and command you posted in the issue description.

**Comment by pantharshit00** [comment]
I can reproduce this. 

Right now we are generating a schema like so with the minimal option(Just the schema one). 
```prisma
datasource db {
  provider = "sqlite"
  url      = "sqlite://"
}

generator photon {
  provider = "photonjs"
}

model User {
  id    String  @default(cuid()) @id @unique
  email String  @unique
  name  String?
  posts Post[]
}

model Post {
  id        String   @default(cuid()) @id @unique
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  published Boolean
  title     String
  content   String?
  author    User?
}
```

It is invalid and out of date for preview019(You can't use `@id` with `@unique` now)



Exact Steps:
1. prisma2 init
2. Blank Project
3. Sqlite
4. Select both photon and lift
5. Typescript
6. Just the schema

**Comment by janpio** [comment]
@pantharshit00 Please create a proper issue for that problem then, if `prisma2 dev` is stumbling in the described way (migration engine) over something like that then this is a Lift problem, not a CLI init one.

**Comment by pantharshit00** [comment]
Opened https://github.com/prisma/prisma2/issues/1237

**Comment by divyenduz** [comment]
Closing this one as #1237 + https://github.com/prisma/prisma2/issues/1295 cover it

This is a duplicate, while #1237 is fixed in alpha, please track #1295 for a resolution for making create working with SQLite3.

If you think that this is a different issue, please comment here and tag me. Thanks.
